src/background/services/openPayments.ts

```diff
@@ -787,6 +787,11 @@ export const isInvalidClientError = (error: any) => {
   return error.status === 400 && error.code === 'invalid_client';
 };
 
+export const isInvalidContinuationError = (error: any) => {
+  if (!isOpenPaymentsClientError(error)) return false;
+  return error.status === 401 && error.code === 'invalid_continuation';
```

Comment:
Based on what I see in the code, when revoking/cancelling grant, we should not end up returning this error. 
`invalid_continuation` only happens on grant continuation

Reply:
It is infact happening as seen here: https://github.com/interledger/web-monetization-extension/issues/595#issuecomment-2355760034 when trying to cancel grant due to `getByContinue` being used here:

https://github.com/interledger/rafiki/blob/9a7eed9114ac9cc4cd412e6d5e8c8af7b3879a55/packages/auth/src/signature/middleware.ts#L90-L101
+
https://github.com/interledger/rafiki/blob/9a7eed9114ac9cc4cd412e6d5e8c8af7b3879a55/packages/auth/src/app.ts#L288-L297

Reply:
Excuses, yes, you are correct :)